cvr/src/main/java/org/cbioportal/cmo/pipelines/CVRPipeline.java

```diff
@@ -136,6 +136,7 @@ else if (gml) {
             checkExceptions(jobExecution, jobParameters, emailUtil);
         }
         log.info("Shutting down CVR Pipeline");
+        System.exit(SpringApplication.exit(ctx));
```

Comment:
added here and [here](https://github.com/knowledgesystems/cmo-pipelines/pull/943/files#diff-4c2f1e731a674f231813db333141c7e462a8c3dddbd610fc26d98ccef29e633dR172) because the application wasn't closing on its own